src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerFileReference.cs

```diff
@@ -622,6 +624,27 @@ private ImmutableArray<TExtension> GetAnalyzersForTypeNames(Assembly analyzerAss
                 }
 
                 return analyzers.ToImmutable();
+
+                static IEnumerable<string> shuffle(ImmutableSortedSet<string> source)
+                {
+                    var random =
+#if NET6_0_OR_GREATER
+                        Random.Shared;
+#else
+                new Random();
```

Comment:
Is the spacing different between TFMs here? Or is it github messing something up?

Reply:
Hmm, fixed